src/main.cpp

```diff
@@ -14,4 +15,13 @@ int main() {
     // Literal syntax
     auto a1 = "a1"_sq;
     uint64_t board = 1ULL << static_cast<int>(a1);
```

Comment:
Change `uint64_t` to `Bitboard`. Even though the types _do_ technically work out, it looks weird in its current form.

Reply:
this is what you had in main. I just kept it as it is. my board is initialised as Bitboard

Reply:
I am aware of that. You should change it anyways.

Reply:
done